compiler+runtime/src/cpp/jank/runtime/detail/object_util.cpp

```diff
@@ -7,6 +7,11 @@ namespace jank::runtime::detail
     return visit_object([](auto const typed_o) { return typed_o->to_string(); }, o);
   }
 
+  void to_string(char const ch, fmt::memory_buffer &buff)
+  {
+    make_box(ch)->to_string(buff);
```

Comment:
There is no need to do an allocation here. If you need a `character` temporarily, just create one locally.

```cpp
obj::character{ ch }.to_string(buff);
```